command/agent/agent.go

```diff
@@ -115,7 +110,7 @@ func (a *Agent) serverConfig() (*nomad.Config, error) {
 		if a.config.Server.BootstrapExpect == 1 {
 			conf.Bootstrap = true
 		} else {
-			atomic.StoreInt32(&conf.BootstrapExpect, int32(a.config.Server.BootstrapExpect))
+			conf.BootstrapExpect = int32(a.config.Server.BootstrapExpect)
```

Comment:
I would just use the atomics. It is easier to just know that all users of the atomic variable are accessing it atomically